Kernel/Arch/aarch64/Registers.h

```diff
@@ -454,6 +454,54 @@ struct alignas(u64) SCR_EL3 {
 };
 static_assert(sizeof(SCR_EL3) == 8);
 
+// C5.2.18 SPSR_EL1, Saved Program Status Register (EL1)
+struct alignas(u64) SPSR_EL1 {
+    enum Mode : u8 {
+        EL0t = 0b0000,
+        EL1t = 0b0100,
+        EL1h = 0b0101
+    };
+
+    Mode M : 4;
+    int M_4 : 1 = 0;
+    int _reserved5 : 1 = 0;
```

Comment:
(no need to put names here probably)

Reply:
Oh, I wasn't actually aware that that was even possible. I followed the convention of other structs in this file, which also use `_reservedx`. However it looks like other structs actually don't put a name for the fields, so this can be cleaned up in the future indeed :)